.github/workflows/deploy-book.yml

```
@@ -4,7 +4,7 @@ name: deploy-book
 # Only run this when the main branch changes
 on:
   # Uncomment the 'pull_request' line below to manually re-build Jupyter Book
-  # pull_request:
+  pull_request:
```

TODO comment this out before merging. Just having this run in the PR so things are rendered at https://www.generic-mapping-tools.org/egu22pygmt
```suggestion
  # pull_request:
```

---

Done in 5e0006b037b9ce9e166788632fdd6b4e6db6dd1a